cx-server-companion/files/cx-server-companion.sh

```diff
@@ -590,6 +591,11 @@ function start_jenkins_container()
         if [ -e /cx-server/mount/jenkins-configuration ]; then
             environment_variable_parameters+=(-e CASC_JENKINS_CONFIG=/var/cx-server/jenkins-configuration)
         fi
+
+        for var in $(env | grep ^CX) do
+            environment_variable_parameters+=(-e $var)
```

Comment:
I get an syntax error in that line :-(

Reply:
Yes, me too, but I don't get it yet.. What is wrong?

Reply:
```
for x in y
do
      # do something
done
```
or
```
for x in y; do
      # do something
done
```